### PR TITLE
Performance tests documentation and JMeter Test Plan script

### PR DESCRIPTION
**Pull request by lukaszmorawski**
* Test methodology wiki page,
* Test results summary wiki page,
* JMeter Test Plan as .jmx file

### CODE REVIEW

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes methodology of Knotx application performance tests.
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: How about

_This page describes *the* methodology of Knotx performance tests._

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## Goals
+Goals of conducted tests are as following:
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: This sentence is incorrect. Please change it to the following one:

_**The** goals of conducted tests are **as follows**:_

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+* Assess system performance,
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: You seem to be using Polish punctuation rules.

Check out [this article](https://en.oxforddictionaries.com/punctuation/bullet-points) for an overview of punctuation in bullet points.

TL;DR the comma looks strange. You can use periods in each bullet points or just refrain from using punctuation marks at the end of each line.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+* Assess system stability.
+
+### Metrics
+During our tests we focus on following metrics and correlate it to came out with observations:
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: > During our tests we focus on following metrics and correlate it to came out with observations:

This sentence is grammatically incorrect and I'm not sure what you meant to say. I'll be happy to help you rephrase it if drop me a line on Skype.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+* throughput,
+* response time - 90% Line,
+* quantity of Virtual Users,
+* Number of errors (HTTP response codes > 400, Knotx logs),
+* infrastructure resources (cpu, memory, disk, network),
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Awkward, Polish punctuation.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+* 1 x Knotx - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8,
+* 1 x Mocks - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8,
+* 1 x AEM - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth.centos 6.8.
+
+### Test harness
+* 1 x Jmeter Master - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8,
+* 2 x Jmeter Servers - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8.
+
+### Repositories
+* mocks-knotx - https://stash.cognifide.com/projects/OCTPI/repos/mocks-knotx/browse
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: We can't link to cognifide stash here :) we'd need to move it to this repo. e.g. /performance-tests/mocks-knotx and /performance-tests/config-knotx

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+* Test script implemented in JMeter 3.1,
+* Think time 4-6 seconds (Gaussian Random Timer),
+* No embedded resources are downloaded, just HTML,
+* Assertions for each requests to verify correct data is returned (Response Assertion),
+* Test scripts are tested,
+* Load ramp-up for 5400 (1.5 hour)  seconds, then sustain load for another 1 hour.
+
+## Execution
+* Jmeter run in Distribute Mode (Remote),
+* Restart all servers before tests (Apache Front, Knotx, Mocks, JMeter machines),
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: What does it mean "Apache Front" ? Btw. if you're referenting apache http server use the whole name "Apache HTTP Server" as the Apache is foundation name, and HTTP Server is the tool

File: documentation/src/main/wiki/PerformanceTestsSummary.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+This page gathers all tests summaries, recommendations and actions undertaken.
+
+## Overview
+* Throughput of Knotx varies from **223 requests/sec** up to **840 requests/second** for different scenarios for **1 second response time**,
+* Knotx is stable,
+* Knotx recovers after crashes,
+* Several tweaks and fixes introduced to Knotx and Apache HTTP Server.
+
+## Tests Summary
+<p align="center">
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsSummary.md]: Use wiki directives to put images and remove HTML markup, e.g. in this case
```
![Basic request flow](assets/knotx-perf-tests-summary.png)
```

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 4 above)
Comment (toniedzwiedz) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: _During our tests, we focus on the following metrics which, when corellated, allow us to come up with concrete observations about the system:_

File: knotx-performance-tests/knotx.jmx [hunk 9]
```diff
@@ -0,0 +1,1517 @@
...
+        <collectionProp name="Arguments.arguments">
+          <elementProp name="protocol" elementType="Argument">
+            <stringProp name="Argument.name">protocol</stringProp>
+            <stringProp name="Argument.value">http</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+            <stringProp name="Argument.desc">https. http</stringProp>
+          </elementProp>
+          <elementProp name="domain" elementType="Argument">
+            <stringProp name="Argument.name">domain</stringProp>
+            <stringProp name="Argument.value">octopus.cognifide.com</stringProp>
```
Comment (marcinczeczko) [inline knotx-performance-tests/knotx.jmx]: I'd rather put here something that's not cognifide.com, e.g.: knotx.performance.local

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 2 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: fixed

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 3 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: fixed

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 4 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: done

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 5 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: changed

**Comment by toniedzwiedz** [comment]
I don't think we need this file https://github.com/Cognifide/knotx/pull/267/files#diff-2cfe0e7f508bf2e3420ed58fc9c0a18b as

 Subject | Predicate | Object 
---------|----------|--------
GitHub Flavoured Markdown | supports | tables 

```
 Subject | Predicate | Object 
---------|----------|--------
GitHub Flavoured Markdown | supports | tables 
```

Check out the the docs https://guides.github.com/features/mastering-markdown/

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 1 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: fixed

File: knotx-performance-tests/knotx.jmx (hunk 9 above)
Comment (lukaszmorawski) [inline knotx-performance-tests/knotx.jmx]: We conducted tests on Congnifide environment. Now tests are repeatable, you can take it and run it again. Otherwise we need to remember to change it is the scripts.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 10]
```diff
@@ -1,29 +1,29 @@
...
-During our tests we focus on following metrics and correlate it to came out with observations:
-* throughput,
-* response time - 90% Line,
-* quantity of Virtual Users,
-* Number of errors (HTTP response codes > 400, Knotx logs),
-* infrastructure resources (cpu, memory, disk, network),
-* application resources (jmx heap size, garbage collections).
+During our tests, we focus on the following metrics which, when correlated, allow us to come up with number of observations about the system:
+* Throughput.
+* Response time - 90% Line.
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: I may be nitpicking a little but full stops usually follow bullet points that are full sentences :)
Most of these are just single phrases.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 7 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: changed

File: documentation/src/main/wiki/PerformanceTestsSummary.md (hunk 8 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsSummary.md]: changed

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 6 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: added

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 10 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Yes you are :)

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 11]
```diff
@@ -27,26 +27,31 @@ During our tests, we focus on the following metrics which, when correlated, allo
...
+![Test environment](assets/knotx-test-environment.png)
 
 ### System under test
 * 1 x Apache - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
 * 1 x Knot.x - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
 * 1 x Mocks - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
 * 1 x AEM - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth.centos 6.8.
 
+### Knotx author
+java -Dlogback.configurationFile=/content/knotx/author/logback.xml -Dvertx.cacheDirBase=/content/knotx/author/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=18092 -Dcom.sun.management.jmxremote.rmi.port=18092 -Djava.rmi.server.hostname=10.254.165.236 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/author/knotx.jar -conf /content/knotx/author/config/author-config.json
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Consider using preformatted, monospace text for commands

Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Added in last commit.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+	
+## Test Environment
+Test Environment is set up on AWS Cloud.
+![Test environment](assets/knotx-test-environment.png)
+
+### System under test
+* 1 x Apache - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Knot.x - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Mocks - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x AEM - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth.centos 6.8.
```
Comment (Skejven) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: There is no such machine on the test environment structure. Also AEM is not mentioned in tests. Looks like some copy paste?

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+* config-knotx - \knotx-performance-tests\mocks-knotx\
+
+## Tools
+* JMeter 3.1 with plugins (at AWS) as a Load Generator.
+* Zabbix 3.2.1 (Internal instance at Cognifide Office) as an Server Monitoring.
+* Influx (Internal Instance at AWS set up on JMeter Master machine) as a Live Test Monitoring database.
+* Grafana 3.0.1 (Internal Instance at Cognifide Office) as a Live Test Monitoring tool.
+
+## Test Process
+Test process that we used is based on Microsoft Performance testing process - https://msdn.microsoft.com/en-us/library/bb924376.aspx.
```
Comment (Skejven) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: We can have a link here, e.g.:
`[Microsoft Performance testing process](https://msdn.microsoft.com/en-us/library/bb924376.aspx).`

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+
+## Test Scripts
+* Test script implemented in JMeter 3.1.
+* Think time 4-6 seconds (Gaussian Random Timer).
```
Comment (Skejven) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Do you think we should add some definitions or urls to definitions here?
E.g. [Think time](http://www.webperformance.com/library/dictionary.html#thinktime) ?

File: knotx-performance-tests/config-knotx/author-config.json [hunk 15]
```diff
@@ -0,0 +1,182 @@
+{
```
Comment (Skejven) [inline knotx-performance-tests/config-knotx/author-config.json]: Do we need those 2 configurations? Are they used? What is `publish` and what is `author` config?

File: knotx-performance-tests/jmeter-testplan/knotx.jmx [hunk 16]
```diff
@@ -0,0 +1,1517 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Skejven) [inline knotx-performance-tests/jmeter-testplan/knotx.jmx]: We probably should consider adding [Apache License Header](https://github.com/Cognifide/knotx/blob/master/LICENSE#L189) in all `XML` files.
You may take example for XML from [pom.xml](https://github.com/Cognifide/knotx/blob/master/pom.xml#L2).

File: knotx-performance-tests/mocks-knotx/content/simple-no-snippets.html [hunk 17]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (Skejven) [inline knotx-performance-tests/mocks-knotx/content/simple-no-snippets.html]: License header missing. Maybe we don't use this file at all in tests?

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 12 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Removed AEM. It's for future tests.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 13 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: fixed

File: knotx-performance-tests/config-knotx/author-config.json (hunk 15 above)
Comment (lukaszmorawski) [inline knotx-performance-tests/config-knotx/author-config.json]: We did not used and tested author but I moved whole configuration to have it documented.

File: knotx-performance-tests/jmeter-testplan/knotx.jmx (hunk 16 above)
Comment (lukaszmorawski) [inline knotx-performance-tests/jmeter-testplan/knotx.jmx]: knotx.jmx - this is JMeter Test Plan file. It is generated by JMeter and we do not change it. Jmeter is under Apache License and developed under Apache Foundation.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 18]
```diff
@@ -0,0 +1,99 @@
...
+    * No errors (logs, responses).
+    * Infrastructure resources OK.
+    * Application resources OK.
+	
+## Test Environment
+Test Environment is set up on AWS Cloud.
+![Test environment](assets/knotx-test-environment.png)
+
+### System under test
+* 1 x Apache - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Apache HTTP Server

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 19]
```diff
@@ -0,0 +1,99 @@
...
+* 1 x Knot.x - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Mocks - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Repository & Services Mocks

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 12 above)
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: I don't see it's removed

File: knotx-performance-tests/mocks-knotx/content/simple-no-snippets.html (hunk 17 above)
Comment (lukaszmorawski) [inline knotx-performance-tests/mocks-knotx/content/simple-no-snippets.html]: Yes, we did not use it in our tests. Removed

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 20]
```diff
@@ -0,0 +1,99 @@
...
+* 1 x AEM - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth.centos 6.8.
+
+### Knotx author start command
+`java -Dlogback.configurationFile=/content/knotx/author/logback.xml -Dvertx.cacheDirBase=/content/knotx/author/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=18092 -Dcom.sun.management.jmxremote.rmi.port=18092 -Djava.rmi.server.hostname=10.254.165.236 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/author/knotx.jar -conf /content/knotx/author/config/author-config.json`
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: It's really hard to see what's being set in those command, instead consider using code snippet as follows:
```
java -Dlogback.configurationFile=/content/knotx/author/logback.xml \
     -Dvertx.cacheDirBase=/content/knotx/author/.vertx \
     -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m \
     -XX:ReservedCodeCacheSize=128m \
     -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=18092 \
     ...
```
Additionally, I'd modify hostname IPs to something more generic (e.g. rmi.server.hostname)

This should be applied to remaining commands too

File: documentation/src/main/wiki/PerformanceTestsMethodology.md [hunk 21]
```diff
@@ -0,0 +1,99 @@
...
+
+### Knotx mocks start command
+`java -Dlogback.configurationFile=/content/knotx/mocks/logback.xml -Dvertx.cacheDirBase=/content/knotx/mocks/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=19000 -Dcom.sun.management.jmxremote.rmi.port=19000 -Djava.rmi.server.hostname=10.254.165.163 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/mocks/knotx.jar -conf /content/knotx/mocks/config/mocks-config.json`
+
+### Test harness
+* 1 x Jmeter Master - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8.
+* 2 x Jmeter Servers - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8.
+
+### Repositories
+* mocks-knotx - \knotx-performance-tests\config-knotx\
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Make full link to the folder on github, e.g. using markdown as below
```
* mocks-knot - [knotx-perfromance-tests\config-knotx](https://github.com/Cognifide/knotx/tree/master/knotx-perfromance-tests/config-knotx)
```

Same for next bullet

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 13 above)
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: I don't see fixes for this

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 14 above)
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: 👍

File: documentation/src/main/wiki/PerformanceTestsTuning.md [hunk 22]
```diff
@@ -0,0 +1,41 @@
+# Tuning
+
+## Introduction
+During performance tests we've realized that default configurations are a bottleneck. This applies to Apache HTTP Server, Knot.x and Mocks configuration. To move forward, we've introduced following tuning.
+
+## Apache tuning
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsTuning.md]: Apache HTTP Server

File: documentation/src/main/wiki/PerformanceTestsTuning.md [hunk 23]
```diff
@@ -0,0 +1,41 @@
...
+  ThreadsPerChild      64
+  MaxRequestsPerChild  0
+  ListenBackLog        511
+</IfModule>
+```
+
+## Apache ulimit
+`apache - nofile 16384`
+
+## Sysctl tuning on knotx, mocks and apache servers
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsTuning.md]:
```
## Sysctl tuning on Knot.x, Mocks and Apache HTTP Servers
```

File: documentation/src/main/wiki/PerformanceTestsSummary.md [hunk 24]
```diff
@@ -0,0 +1,13 @@
+# Performance Tests Summary
+
+## Introduction
+This page gathers all tests summaries, recommendations and actions undertaken.
+
+## Overview
+* Throughput of Knot.x varies from **223 requests/sec** up to **840 requests/second** for different scenarios for **1 second response time**.
+* Knot.x is stable.
+* Knot.x recovers after crashes.
+* Several tweaks and fixes introduced to [[Knot.x, Mocks and Apache HTTP Server|PerformanceTestsTuning]]
```
Comment (marcinczeczko) [inline documentation/src/main/wiki/PerformanceTestsSummary.md]: `* Several fixes applied to Knot.x as well as configuration tweaks both for Knot.x and Apache HTTP Server`
Just to state clearly we didn't fixed Apache HTTP but we fixed configuration of it.

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 14 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Terminology links added

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 18 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Changed

File: documentation/src/main/wiki/PerformanceTestsMethodology.md (hunk 19 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsMethodology.md]: Changed

File: documentation/src/main/wiki/PerformanceTestsSummary.md (hunk 24 above)
Comment (lukaszmorawski) [inline documentation/src/main/wiki/PerformanceTestsSummary.md]: done

File: knotx-performance-tests/config-knotx/author-config.json (hunk 15 above)
Comment (tomaszmichalak) [inline knotx-performance-tests/config-knotx/author-config.json]: Fixed.
